pyinfraformat/core/core.py

```diff
@@ -308,6 +308,36 @@ def to_infraformat(self, path, split=False, namelist=None):
         else:
             to_infraformat(self.holes, path)
 
+    def project(self, output="EPSG:4326", check="Finland", output_height=False):
```

Comment:
```suggestion
    def project(self, output="EPSG:3857", check="Finland", output_height=False):
```

Reply:
Last comment. And i dont think anyone would want to project holes into Web Mercator, which is only used by tileproviders.